addons/website/static/src/snippets/s_dynamic_snippet/options.js

```diff
@@ -61,6 +61,18 @@ const dynamicSnippetOptions = options.Class.extend({
     //--------------------------------------------------------------------------
     // Public
     //--------------------------------------------------------------------------
+    /**
+     * See from updateUI in s_website_form
+     * @override
+     */
```

Comment:
- Add a blank line between the public section title and the documentation of this function
- Add a blank line between the description and the `@override`